test/bigtest/tests/login-test.js

```diff
@@ -19,13 +27,355 @@ describe('Login', () => {
     expect(login.submit.isDisabled).to.be.true;
   });
 
-  describe('with invalid credentials', () => {
-    it.always('does not log in');
-    it('displays an error');
+  it('error message should not be present', () => {
```

Comment:
When we test that something _should not_ happen, there can be times where the test passes before the thing happens. We can use `it.always` here to ensure that this test passes for a short period of time (`200ms` by default).

You can learn more about these type of tests here: [@bigtest/mocha](https://github.com/bigtestjs/mocha/blob/master/README.md)